src/v2/Apps/Show/Components/ShowViewingRoom.tsx

```diff
@@ -1,22 +1,82 @@
 import React from "react"
-import { Box, BoxProps, ResponsiveBox, Text } from "@artsy/palette"
+import { Box, BoxProps, MediumCard, ResponsiveBox, Text } from "@artsy/palette"
+import { createFragmentContainer, graphql } from "react-relay"
+import { ShowViewingRoom_show } from "v2/__generated__/ShowViewingRoom_show.graphql"
+import { RouterLink } from "v2/Artsy/Router/RouterLink"
+import { crop } from "v2/Utils/resizer"
+import { getTagProps } from "v2/Components/ViewingRoomCard"
 
-interface ShowViewingRoom extends BoxProps {}
+interface ShowViewingRoomProps extends BoxProps {
+  show: ShowViewingRoom_show
+}
+
+export const ShowViewingRoom: React.FC<ShowViewingRoomProps> = ({
+  show,
+  ...rest
+}) => {
+  const [{ node: viewingRoom }] = show.viewingRoomsConnection.edges
```

Comment:
oo i didnt know about this kind of destructuring!

Reply:
Since we just want the first one — note there's no per/size/limit on the viewingRooms connection which maybe there should be